streaming/index.js

```diff
@@ -537,12 +561,15 @@ const startServer = async () => {
     req.chosenLanguages = result.rows[0].chosen_languages;
     req.deviceId = result.rows[0].device_id;
 
+    req.cachedFilters = await fetchFilters(req.accountId);
```

Comment:
For now I've left this on the `req` object, though, in the future we probably want to decouple from the `req` object to something with a different structure (this is based on some recent explorations of how we can further clean up this code)